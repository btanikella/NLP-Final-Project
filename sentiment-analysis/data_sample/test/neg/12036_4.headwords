time
has
are
bad
plays
product
's
devoid
has
