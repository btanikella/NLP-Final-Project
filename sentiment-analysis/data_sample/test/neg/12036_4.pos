NNP  NNP  NNP NNP VBZ DT JJ NN NN
PRP VBZ DT RB JJ NN  DT NN VBZ RB JJ
EX VBP DT NN IN NNS
CC DT NN NNS VBP RB JJ
CC IN DT NN  PRP VBZ IN DT JJ NN NN IN DT NNS IN DT NN VBN IN
DT NN VBZ DT NN NN IN DT  NN  VBG TO VB DT RB JJ NN
EX VBZ RB JJ NN RB CC DT NN VBZ RB JJ NN IN VBZ CC NN NNS
IN IN DT NN IN DT NN IN DT JJ NN  NN VBZ RB JJ IN DT NNS CC NN CC NN
DT NN VBZ RB JJ NN IN DT NN NN POS JJ RB IN PRP VBZ RB VB DT JJ NN
