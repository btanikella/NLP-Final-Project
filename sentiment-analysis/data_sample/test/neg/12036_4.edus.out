Sentence #1 (12 tokens):
MARY, MARY, BLOODY MARY is an OK time killer.
[Text=MARY CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=8 CharacterOffsetEnd=9 PartOfSpeech=,] [Text=MARY CharacterOffsetBegin=10 CharacterOffsetEnd=14 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=14 CharacterOffsetEnd=15 PartOfSpeech=,] [Text=BLOODY CharacterOffsetBegin=16 CharacterOffsetEnd=22 PartOfSpeech=NNP] [Text=MARY CharacterOffsetBegin=23 CharacterOffsetEnd=27 PartOfSpeech=NNP] [Text=is CharacterOffsetBegin=28 CharacterOffsetEnd=30 PartOfSpeech=VBZ] [Text=an CharacterOffsetBegin=31 CharacterOffsetEnd=33 PartOfSpeech=DT] [Text=OK CharacterOffsetBegin=34 CharacterOffsetEnd=36 PartOfSpeech=JJ] [Text=time CharacterOffsetBegin=37 CharacterOffsetEnd=41 PartOfSpeech=NN] [Text=killer CharacterOffsetBegin=42 CharacterOffsetEnd=48 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=48 CharacterOffsetEnd=49 PartOfSpeech=.] 
root(ROOT-0, time-10)
nsubj(time-10, MARY-1)
punct(MARY-1, ,-2)
conj(MARY-1, MARY-3)
punct(MARY-1, ,-4)
nn(MARY-6, BLOODY-5)
appos(MARY-1, MARY-6)
cop(time-10, is-7)
det(time-10, an-8)
amod(time-10, OK-9)
dep(time-10, killer-11)
punct(time-10, .-12)

Sentence #2 (13 tokens):
It has a uniformly attractive cast, the action is rarely dull.
[Text=It CharacterOffsetBegin=54 CharacterOffsetEnd=56 PartOfSpeech=PRP] [Text=has CharacterOffsetBegin=57 CharacterOffsetEnd=60 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=61 CharacterOffsetEnd=62 PartOfSpeech=DT] [Text=uniformly CharacterOffsetBegin=63 CharacterOffsetEnd=72 PartOfSpeech=RB] [Text=attractive CharacterOffsetBegin=73 CharacterOffsetEnd=83 PartOfSpeech=JJ] [Text=cast CharacterOffsetBegin=84 CharacterOffsetEnd=88 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=88 CharacterOffsetEnd=89 PartOfSpeech=,] [Text=the CharacterOffsetBegin=90 CharacterOffsetEnd=93 PartOfSpeech=DT] [Text=action CharacterOffsetBegin=94 CharacterOffsetEnd=100 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=101 CharacterOffsetEnd=103 PartOfSpeech=VBZ] [Text=rarely CharacterOffsetBegin=104 CharacterOffsetEnd=110 PartOfSpeech=RB] [Text=dull CharacterOffsetBegin=111 CharacterOffsetEnd=115 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=115 CharacterOffsetEnd=116 PartOfSpeech=.] 
root(ROOT-0, has-2)
nsubj(has-2, It-1)
det(cast-6, a-3)
advmod(attractive-5, uniformly-4)
amod(cast-6, attractive-5)
dobj(has-2, cast-6)
punct(has-2, ,-7)
det(action-9, the-8)
nsubj(dull-12, action-9)
cop(dull-12, is-10)
advmod(dull-12, rarely-11)
dep(has-2, dull-12)
punct(has-2, .-13)

Sentence #3 (7 tokens):
There are a lot of killings.
[Text=There CharacterOffsetBegin=121 CharacterOffsetEnd=126 PartOfSpeech=EX] [Text=are CharacterOffsetBegin=127 CharacterOffsetEnd=130 PartOfSpeech=VBP] [Text=a CharacterOffsetBegin=131 CharacterOffsetEnd=132 PartOfSpeech=DT] [Text=lot CharacterOffsetBegin=133 CharacterOffsetEnd=136 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=137 CharacterOffsetEnd=139 PartOfSpeech=IN] [Text=killings CharacterOffsetBegin=140 CharacterOffsetEnd=148 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=148 CharacterOffsetEnd=149 PartOfSpeech=.] 
root(ROOT-0, are-2)
expl(are-2, There-1)
det(lot-4, a-3)
nsubj(are-2, lot-4)
prep(lot-4, of-5)
pobj(of-5, killings-6)
punct(are-2, .-7)

Sentence #4 (8 tokens):
And the production values are not bad.
[Text=And CharacterOffsetBegin=154 CharacterOffsetEnd=157 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=158 CharacterOffsetEnd=161 PartOfSpeech=DT] [Text=production CharacterOffsetBegin=162 CharacterOffsetEnd=172 PartOfSpeech=NN] [Text=values CharacterOffsetBegin=173 CharacterOffsetEnd=179 PartOfSpeech=NNS] [Text=are CharacterOffsetBegin=180 CharacterOffsetEnd=183 PartOfSpeech=VBP] [Text=not CharacterOffsetBegin=184 CharacterOffsetEnd=187 PartOfSpeech=RB] [Text=bad CharacterOffsetBegin=188 CharacterOffsetEnd=191 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=191 CharacterOffsetEnd=192 PartOfSpeech=.] 
root(ROOT-0, bad-7)
cc(bad-7, And-1)
det(values-4, the-2)
nn(values-4, production-3)
nsubj(bad-7, values-4)
cop(bad-7, are-5)
neg(bad-7, not-6)
punct(bad-7, .-8)

Sentence #5 (21 tokens):
But in the end, it plays like a standard TV episode from the 1970s with some nudity thrown in.
[Text=But CharacterOffsetBegin=197 CharacterOffsetEnd=200 PartOfSpeech=CC] [Text=in CharacterOffsetBegin=201 CharacterOffsetEnd=203 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=204 CharacterOffsetEnd=207 PartOfSpeech=DT] [Text=end CharacterOffsetBegin=208 CharacterOffsetEnd=211 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=211 CharacterOffsetEnd=212 PartOfSpeech=,] [Text=it CharacterOffsetBegin=213 CharacterOffsetEnd=215 PartOfSpeech=PRP] [Text=plays CharacterOffsetBegin=216 CharacterOffsetEnd=221 PartOfSpeech=VBZ] [Text=like CharacterOffsetBegin=222 CharacterOffsetEnd=226 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=227 CharacterOffsetEnd=228 PartOfSpeech=DT] [Text=standard CharacterOffsetBegin=229 CharacterOffsetEnd=237 PartOfSpeech=JJ] [Text=TV CharacterOffsetBegin=238 CharacterOffsetEnd=240 PartOfSpeech=NN] [Text=episode CharacterOffsetBegin=241 CharacterOffsetEnd=248 PartOfSpeech=NN] [Text=from CharacterOffsetBegin=249 CharacterOffsetEnd=253 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=254 CharacterOffsetEnd=257 PartOfSpeech=DT] [Text=1970s CharacterOffsetBegin=258 CharacterOffsetEnd=263 PartOfSpeech=NNS] [Text=with CharacterOffsetBegin=264 CharacterOffsetEnd=268 PartOfSpeech=IN] [Text=some CharacterOffsetBegin=269 CharacterOffsetEnd=273 PartOfSpeech=DT] [Text=nudity CharacterOffsetBegin=274 CharacterOffsetEnd=280 PartOfSpeech=NN] [Text=thrown CharacterOffsetBegin=281 CharacterOffsetEnd=287 PartOfSpeech=VBN] [Text=in CharacterOffsetBegin=288 CharacterOffsetEnd=290 PartOfSpeech=IN] [Text=. CharacterOffsetBegin=290 CharacterOffsetEnd=291 PartOfSpeech=.] 
root(ROOT-0, plays-7)
cc(plays-7, But-1)
prep(plays-7, in-2)
det(end-4, the-3)
pobj(in-2, end-4)
punct(plays-7, ,-5)
nsubj(plays-7, it-6)
prep(plays-7, like-8)
det(episode-12, a-9)
amod(episode-12, standard-10)
nn(episode-12, TV-11)
pobj(like-8, episode-12)
prep(episode-12, from-13)
det(1970s-15, the-14)
pobj(from-13, 1970s-15)
prep(episode-12, with-16)
det(nudity-18, some-17)
pobj(with-16, nudity-18)
partmod(nudity-18, thrown-19)
prep(thrown-19, in-20)
punct(plays-7, .-21)

Sentence #6 (19 tokens):
The film is the end product of an "author" trying to make a purely commercial film.
[Text=The CharacterOffsetBegin=296 CharacterOffsetEnd=299 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=300 CharacterOffsetEnd=304 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=305 CharacterOffsetEnd=307 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=308 CharacterOffsetEnd=311 PartOfSpeech=DT] [Text=end CharacterOffsetBegin=312 CharacterOffsetEnd=315 PartOfSpeech=NN] [Text=product CharacterOffsetBegin=316 CharacterOffsetEnd=323 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=324 CharacterOffsetEnd=326 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=327 CharacterOffsetEnd=329 PartOfSpeech=DT] [Text=`` CharacterOffsetBegin=330 CharacterOffsetEnd=331 PartOfSpeech=``] [Text=author CharacterOffsetBegin=331 CharacterOffsetEnd=337 PartOfSpeech=NN] [Text='' CharacterOffsetBegin=337 CharacterOffsetEnd=338 PartOfSpeech=''] [Text=trying CharacterOffsetBegin=339 CharacterOffsetEnd=345 PartOfSpeech=VBG] [Text=to CharacterOffsetBegin=346 CharacterOffsetEnd=348 PartOfSpeech=TO] [Text=make CharacterOffsetBegin=349 CharacterOffsetEnd=353 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=354 CharacterOffsetEnd=355 PartOfSpeech=DT] [Text=purely CharacterOffsetBegin=356 CharacterOffsetEnd=362 PartOfSpeech=RB] [Text=commercial CharacterOffsetBegin=363 CharacterOffsetEnd=373 PartOfSpeech=JJ] [Text=film CharacterOffsetBegin=374 CharacterOffsetEnd=378 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=378 CharacterOffsetEnd=379 PartOfSpeech=.] 
root(ROOT-0, product-6)
det(film-2, The-1)
nsubj(product-6, film-2)
cop(product-6, is-3)
det(product-6, the-4)
nn(product-6, end-5)
prep(product-6, of-7)
det(author-10, an-8)
punct(author-10, ``-9)
pobj(of-7, author-10)
punct(author-10, ''-11)
partmod(author-10, trying-12)
aux(make-14, to-13)
xcomp(trying-12, make-14)
det(film-18, a-15)
amod(film-18, purely-16)
amod(film-18, commercial-17)
dobj(make-14, film-18)
punct(product-6, .-19)

Sentence #7 (19 tokens):
There's very little depth here and the film spends too much time with chases and action scenes.
[Text=There CharacterOffsetBegin=384 CharacterOffsetEnd=389 PartOfSpeech=EX] [Text='s CharacterOffsetBegin=389 CharacterOffsetEnd=391 PartOfSpeech=VBZ] [Text=very CharacterOffsetBegin=392 CharacterOffsetEnd=396 PartOfSpeech=RB] [Text=little CharacterOffsetBegin=397 CharacterOffsetEnd=403 PartOfSpeech=JJ] [Text=depth CharacterOffsetBegin=404 CharacterOffsetEnd=409 PartOfSpeech=NN] [Text=here CharacterOffsetBegin=410 CharacterOffsetEnd=414 PartOfSpeech=RB] [Text=and CharacterOffsetBegin=415 CharacterOffsetEnd=418 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=419 CharacterOffsetEnd=422 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=423 CharacterOffsetEnd=427 PartOfSpeech=NN] [Text=spends CharacterOffsetBegin=428 CharacterOffsetEnd=434 PartOfSpeech=VBZ] [Text=too CharacterOffsetBegin=435 CharacterOffsetEnd=438 PartOfSpeech=RB] [Text=much CharacterOffsetBegin=439 CharacterOffsetEnd=443 PartOfSpeech=JJ] [Text=time CharacterOffsetBegin=444 CharacterOffsetEnd=448 PartOfSpeech=NN] [Text=with CharacterOffsetBegin=449 CharacterOffsetEnd=453 PartOfSpeech=IN] [Text=chases CharacterOffsetBegin=454 CharacterOffsetEnd=460 PartOfSpeech=VBZ] [Text=and CharacterOffsetBegin=461 CharacterOffsetEnd=464 PartOfSpeech=CC] [Text=action CharacterOffsetBegin=465 CharacterOffsetEnd=471 PartOfSpeech=NN] [Text=scenes CharacterOffsetBegin=472 CharacterOffsetEnd=478 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=478 CharacterOffsetEnd=479 PartOfSpeech=.] 
root(ROOT-0, 's-2)
expl('s-2, There-1)
advmod(little-4, very-3)
amod(depth-5, little-4)
nsubj('s-2, depth-5)
advmod('s-2, here-6)
cc('s-2, and-7)
det(film-9, the-8)
nsubj(spends-10, film-9)
conj('s-2, spends-10)
advmod(much-12, too-11)
amod(time-13, much-12)
dobj(spends-10, time-13)
prep(spends-10, with-14)
nn(scenes-18, chases-15)
cc(chases-15, and-16)
conj(chases-15, action-17)
pobj(with-14, scenes-18)
punct('s-2, .-19)

Sentence #8 (24 tokens):
Except for the scene on the beach with the old man, MMBM is almost devoid of any scares or suspense or dread.
[Text=Except CharacterOffsetBegin=484 CharacterOffsetEnd=490 PartOfSpeech=IN] [Text=for CharacterOffsetBegin=491 CharacterOffsetEnd=494 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=495 CharacterOffsetEnd=498 PartOfSpeech=DT] [Text=scene CharacterOffsetBegin=499 CharacterOffsetEnd=504 PartOfSpeech=NN] [Text=on CharacterOffsetBegin=505 CharacterOffsetEnd=507 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=508 CharacterOffsetEnd=511 PartOfSpeech=DT] [Text=beach CharacterOffsetBegin=512 CharacterOffsetEnd=517 PartOfSpeech=NN] [Text=with CharacterOffsetBegin=518 CharacterOffsetEnd=522 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=523 CharacterOffsetEnd=526 PartOfSpeech=DT] [Text=old CharacterOffsetBegin=527 CharacterOffsetEnd=530 PartOfSpeech=JJ] [Text=man CharacterOffsetBegin=531 CharacterOffsetEnd=534 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=534 CharacterOffsetEnd=535 PartOfSpeech=,] [Text=MMBM CharacterOffsetBegin=536 CharacterOffsetEnd=540 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=541 CharacterOffsetEnd=543 PartOfSpeech=VBZ] [Text=almost CharacterOffsetBegin=544 CharacterOffsetEnd=550 PartOfSpeech=RB] [Text=devoid CharacterOffsetBegin=551 CharacterOffsetEnd=557 PartOfSpeech=JJ] [Text=of CharacterOffsetBegin=558 CharacterOffsetEnd=560 PartOfSpeech=IN] [Text=any CharacterOffsetBegin=561 CharacterOffsetEnd=564 PartOfSpeech=DT] [Text=scares CharacterOffsetBegin=565 CharacterOffsetEnd=571 PartOfSpeech=NNS] [Text=or CharacterOffsetBegin=572 CharacterOffsetEnd=574 PartOfSpeech=CC] [Text=suspense CharacterOffsetBegin=575 CharacterOffsetEnd=583 PartOfSpeech=NN] [Text=or CharacterOffsetBegin=584 CharacterOffsetEnd=586 PartOfSpeech=CC] [Text=dread CharacterOffsetBegin=587 CharacterOffsetEnd=592 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=592 CharacterOffsetEnd=593 PartOfSpeech=.] 
root(ROOT-0, devoid-16)
prep(devoid-16, Except-1)
pcomp(Except-1, for-2)
det(scene-4, the-3)
pobj(for-2, scene-4)
prep(scene-4, on-5)
det(beach-7, the-6)
pobj(on-5, beach-7)
prep(scene-4, with-8)
det(man-11, the-9)
amod(man-11, old-10)
pobj(with-8, man-11)
punct(devoid-16, ,-12)
nsubj(devoid-16, MMBM-13)
cop(devoid-16, is-14)
advmod(devoid-16, almost-15)
prep(devoid-16, of-17)
det(scares-19, any-18)
pobj(of-17, scares-19)
cc(scares-19, or-20)
conj(scares-19, suspense-21)
cc(scares-19, or-22)
conj(scares-19, dread-23)
punct(devoid-16, .-24)

Sentence #9 (22 tokens):
The director has very little understanding of the horror genre.It's watchable even though it doesn't leave a lasting impression.
[Text=The CharacterOffsetBegin=598 CharacterOffsetEnd=601 PartOfSpeech=DT] [Text=director CharacterOffsetBegin=602 CharacterOffsetEnd=610 PartOfSpeech=NN] [Text=has CharacterOffsetBegin=611 CharacterOffsetEnd=614 PartOfSpeech=VBZ] [Text=very CharacterOffsetBegin=615 CharacterOffsetEnd=619 PartOfSpeech=RB] [Text=little CharacterOffsetBegin=620 CharacterOffsetEnd=626 PartOfSpeech=JJ] [Text=understanding CharacterOffsetBegin=627 CharacterOffsetEnd=640 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=641 CharacterOffsetEnd=643 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=644 CharacterOffsetEnd=647 PartOfSpeech=DT] [Text=horror CharacterOffsetBegin=648 CharacterOffsetEnd=654 PartOfSpeech=NN] [Text=genre.It CharacterOffsetBegin=655 CharacterOffsetEnd=663 PartOfSpeech=NN] [Text='s CharacterOffsetBegin=663 CharacterOffsetEnd=665 PartOfSpeech=POS] [Text=watchable CharacterOffsetBegin=666 CharacterOffsetEnd=675 PartOfSpeech=JJ] [Text=even CharacterOffsetBegin=676 CharacterOffsetEnd=680 PartOfSpeech=RB] [Text=though CharacterOffsetBegin=681 CharacterOffsetEnd=687 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=688 CharacterOffsetEnd=690 PartOfSpeech=PRP] [Text=does CharacterOffsetBegin=691 CharacterOffsetEnd=695 PartOfSpeech=VBZ] [Text=n't CharacterOffsetBegin=695 CharacterOffsetEnd=698 PartOfSpeech=RB] [Text=leave CharacterOffsetBegin=699 CharacterOffsetEnd=704 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=705 CharacterOffsetEnd=706 PartOfSpeech=DT] [Text=lasting CharacterOffsetBegin=707 CharacterOffsetEnd=714 PartOfSpeech=JJ] [Text=impression CharacterOffsetBegin=715 CharacterOffsetEnd=725 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=725 CharacterOffsetEnd=726 PartOfSpeech=.] 
root(ROOT-0, has-3)
det(director-2, The-1)
nsubj(has-3, director-2)
advmod(little-5, very-4)
amod(understanding-6, little-5)
dobj(has-3, understanding-6)
prep(understanding-6, of-7)
det(genre.It-10, the-8)
nn(genre.It-10, horror-9)
poss(watchable-12, genre.It-10)
possessive(genre.It-10, 's-11)
pobj(of-7, watchable-12)
advmod(leave-18, even-13)
mark(leave-18, though-14)
nsubj(leave-18, it-15)
aux(leave-18, does-16)
neg(leave-18, n't-17)
advcl(has-3, leave-18)
det(impression-21, a-19)
amod(impression-21, lasting-20)
dobj(leave-18, impression-21)
punct(has-3, .-22)

